src/vs/workbench/common/theme.ts

```diff
@@ -138,6 +138,11 @@ export const TAB_UNFOCUSED_INACTIVE_FOREGROUND = registerColor('tab.unfocusedIna
 	hc: Color.white
 }, nls.localize('tabUnfocusedInactiveForeground', "Inactive tab foreground color in an unfocused group. Tabs are the containers for editors in the editor area. Multiple tabs can be opened in one editor group. There can be multiple editor groups."));
 
+export const TAB_UNFOCUSED_ACTIVE_BACKGROUND = registerColor('tab.unfocusedActiveBackground', {
```

Comment:
@dmolin can we do 2 changes here:
* the default colors (for dark, light, hc) should really be `TAB_ACTIVE_BACKGROUND` to preserve todays behaviour
* move this right below the definition of `TAB_ACTIVE_BACKGROUND`

Reply:
done in commit 82f5ef9